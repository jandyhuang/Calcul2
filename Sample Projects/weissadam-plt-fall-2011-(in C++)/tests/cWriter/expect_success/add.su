function iAdd[int a, int b] returns int
	{
	return a + b;
	}

function dAdd[float a, float b] returns float
	{
	return a + b;
	}
function main[] returns int
	{

	int a = 5;
	int b = 2;
	float c = 2.5;
	float d = 5.0;

	print("5 + 2 = ");
	print(iAdd(a,b));

	print("2.5 + 5.0 = ");
	print(dAdd(c,d));

	print("2.5 + 5");
	print(c+a);

	return 0;
	}

