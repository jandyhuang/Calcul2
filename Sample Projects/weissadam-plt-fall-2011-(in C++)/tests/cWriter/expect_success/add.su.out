5 + 2 = 
7
2.5 + 5.0 = 
7.5
2.5 + 5
7.5
